src/js/components/tasks/cards/card.tsx

```diff
@@ -168,27 +203,34 @@ const OrgCard = ({
                   ownedByCurrentUser={ownedByCurrentUser}
                   assignedToCurrentUser={assignedToCurrentUser}
                   ownedByWrongUser={ownedByWrongUser}
+                  reviewOrgOutOfDate={reviewOrgOutOfDate}
                   isCreating={isCreating}
                   isDeleting={isDeleting}
+                  isRefreshingOrg={isRefreshingOrg}
                   doCreateOrg={doCreateOrg}
                   doDeleteOrg={doDeleteOrg}
+                  doRefreshOrg={doRefreshOrg}
                 />
               }
             >
               <OrgInfo
                 org={org}
                 type={type}
+                taskCommits={taskCommits}
+                repoUrl={repoUrl}
                 ownedByCurrentUser={ownedByCurrentUser}
                 assignedToCurrentUser={assignedToCurrentUser}
                 ownedByWrongUser={ownedByWrongUser}
                 isCreating={isCreating}
+                reviewOrgOutOfDate={reviewOrgOutOfDate}
+                missingCommits={orgCommitIdx}
                 doCheckForOrgChanges={doCheckForOrgChanges}
               />
               <OrgSpinner
                 org={org}
                 ownedByCurrentUser={ownedByCurrentUser}
                 isDeleting={isDeleting}
-                isRefreshing={isRefreshing}
+                isRefreshingChanges={isRefreshingChanges}
```

Comment:
This should also get `isRefreshingOrg={isRefreshingOrg}`, handled the same way it handles `isDeleting` or `isRefreshingChanges`.